examples/12. Inline Action.md

```diff
@@ -0,0 +1,66 @@
+FORMAT: 1A
+
+# Inline Action API
```

Comment:
So while we use this name "internally" I do not think we should communicate is as a (new) concept. There is no such a thing as *inlined* transitions or affordances. I would simply name this example as "Advanced Action"